docs/topics/email.txt

```diff
@@ -306,7 +306,8 @@ The class has the following methods:
   Otherwise, an instance of the default backend will be instantiated and
   used. If the keyword argument ``fail_silently`` is ``True``, exceptions
   raised while sending the message will be quashed. An empty list of
-  recipients will not raise an exception.
+  recipients will not raise an exception. It will return ``1`` if
+  successfully delivered the message, otherwise ``0``.
```

Comment:
The sentence is missing some words. I would say:

> It will return `1` if the message was sent successfully, otherwise `0`.

(And "send" parallels the usage earlier in the paragraph.)

Reply:
Sounds better, thanks :+1: